### ISSUE TITLE
Support for tailing logs in kubectl wait

### ISSUE THREAD

**Issue by aleks-m** [kubernetes/kubernetes#100249]
Hi.

I am using kubectl wait to monitor long-running jobs from GitLab CI. Currently I can get logs only after job completion, e.g.:
```
kubectl wait --for=condition=complete --timeout=$KUBECTL_JOB_WAIT_TIMEOUT job/$job_id 
kubectl logs job/$job_id
```
Some of my jobs are taking long time to complete (near 40 mins). So I can't see any log output in that period of time.

Executing `kubectl logs` in my local terminal or looking at logs at a cluster console UI is not an option in my case - my developers have no access to a cluster, I want to implement pure-CI solution.

So it would be nice to be able to use a single command like this instead:
```
kubectl wait --for=condition=complete --logs --timeout=$KUBECTL_JOB_WAIT_TIMEOUT  job/$job_id
```

**Comment by 304NotModified** [comment]
Duplicate of https://github.com/kubernetes/kubernetes/issues/76024
